.github/workflows/isotovideo.yml

```diff
@@ -1,11 +1,48 @@
 ---
 name: isotovideo
 # yamllint disable-line rule:truthy
-on: [push, pull_request]
+on:
+  - pull_request
```

Comment:
Why shouldn't this be run on push?

Reply:
Push means *any push to a branch* regardless of a PR or not. Consumers can change it to their own taste.

Reply:
Yes, but why shouldn't this workflow run when pushed to a branch?

When I want to create a PR, I push to a branch on my fork, and then I expect a workflow already to tell me if there is a problem or not, before I create a PR.
That's useful if I want to experiment first without bothering others with a PR yet.
I don't understand why we wouldn't want that.

Reply:
Hmm. Didn't thought about that use case.

I'd run it locally and then push it to a PR.

Ok. I'll revert that hunk!